### DISCUSSION TITLE
Recruiting for New Ontario Public Health Working Group Proposal

### DISCUSSION THREAD

**Discussion by Asterismm54** [nextgenhealthcare/connect#5340] [General]
Hello All,

This may not be an appropriate place to do this, and I'm open to suggestions for other places!

Back in April 2022, I approached the Ontario Public Health Association about starting a new Working Group ('Work Group') on Open Tools for Public Health.  This would include open standards, open interfaces, open source software, etc. that are relevant to Public Health.

Nothing really happened over the summer, and then on August 4, 2022, I presented a disappointing update to the Ottawa Linux User Group.  They provided very helpful feedback and insights.

On August 5, 2022, I received additional information/approval from the Ontario Public Health Association to move forward with putting together a formal proposal for the formation of the new group.  Now, I am recruiting for reviewers of the proposal.  In addition to the "usual suspects" (Ontario's various constituent public health associations) I am recruiting from some key user and development groups in the relevant communities.  Obviously, Mirth/NextGen users and developers must be given a chance to give feedback on this proposal.

Could I post the proposal here for feedback? Would anyone be interested in providing feedback on this?

Thank you for considering!

**Comment by jonbartels** [comment]
> Could I post the proposal here for feedback? Would anyone be interested in providing feedback on this?

Don't ask to ask, just ask! Edit your post to link to or attach the proposal and see what kind of response you get. 

There is also a user base on the Mirth forums - https://forums.mirthproject.io/ . A user group exists but is somewhat dormant https://www.mcug.org/. The brain trust also spends way too much time on Mirth Slack if some conversational feedback would help.
